lib/misty/http/client.rb

```diff
@@ -67,16 +67,17 @@ def initialize(auth, config, options)
       # and Service API has "/v1/{tenant_id}/stacks"
       # then the path prefix is ignored and path is only "/stacks"
       def self.prefix_path_to_ignore
-        ""
+        ''
```

Comment:
There are different schools for double vs single quotes usage. It seems the trend is to use double. Anyway I think it's important to be consistent and use the same type of approach across the project as we use double.